packages/shared/meta/sagas.ts

```diff
@@ -138,12 +138,15 @@ async function fetchFeatureFlagsAndVariants(network: ETHEREUM_NETWORK): Promise<
   return flagsAndVariants
 }
 
-async function fetchMetaConfiguration(network: ETHEREUM_NETWORK) {
-  const explorerConfigurationEndpoint = getServerConfigurations(network).explorerConfiguration
+async function fetchMetaConfiguration(network: ETHEREUM_NETWORK): Promise<Partial<MetaConfiguration>> {
+  const serverConfiguration = getServerConfigurations(network)
+  const explorerConfigurationEndpoint = serverConfiguration.explorerConfiguration
+  const socialServerUrl = serverConfiguration.socialServerUrl
+
   try {
     const response = await fetch(explorerConfigurationEndpoint)
     if (response.ok) {
-      return response.json()
+      return { ...response.json(), socialServerUrl }
```

Comment:
this can be prevented in the selector. if the response is not OK then it will skip the configuration override entirely

Reply:
moved into selector